app/src/main/java/fr/free/nrw/commons/ViewHolder.java

```diff
@@ -1,7 +1,8 @@
 package fr.free.nrw.commons;
 
-import android.content.Context;
+import fr.free.nrw.commons.contributions.model.DisplayableContribution;
 
 public interface ViewHolder<T> {
-    void bindModel(Context context, T model);
+    void init(int position,
+            DisplayableContribution contribution);
```

Comment:
Thanks for figuring this out, yes I am planning to change it to void bindModel(int position, T model);, context is not required by the view holder, it can get it from its itemView, what do you think ?

Reply:
Yes, I have made ContributionViewHolder not implement the generic ViewHolder, later on when the model holds the position, it could be refactored to implement the ViewHolder